### PR TITLE
Milestone tabale with css created(hard-code)-mobie-v

### PR DESCRIPTION
**Pull request by Zobeir-Rigi**
## Naming Rules

Name your PR like this: ISSUENUMBER-TITLE-YOURNAME

## Description

mobile-v 

## Related to

Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:

Fixes #1

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

File: client/src/component/MilestoneTable.js [hunk 1]
```diff
@@ -1,9 +1,22 @@
-export function MilestoneTable() {
-
+import "../styles/MilestoneTable.css";
+const MilestoneTable = () => {
 	return (
-			<div>
-				<h1>hello world</h1>
-			</div>
+		<div className="table-wrapper">
```
Comment (BoshraM) [inline client/src/component/MilestoneTable.js:4]: i am not sure how we want to name the class we need tp speak about that , for example you use -wraper while I use _container just we need to remind this later to have agreement about for know I think its ok , thank you

File: client/src/component/MilestoneTable.js [hunk 2]
```diff
@@ -1,9 +1,22 @@
...
+				<tbody>
+					<tr>
+						<td>5 out of 7</td>
+						<td>3</td>
+					</tr>
+				</tbody>
+			</table>
+		</div>
 	);
 }
```
Comment (BoshraM) [inline client/src/component/MilestoneTable.js:22]: clean code and nice table , graet

File: client/src/styles/MilestoneTable.css [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    border: 1px solid #808080;
+    background-color: #f2f2f2;
+    padding: 5px;
+}
+td {
+    border: 1px solid #808080;
+    text-align: center;
+    background-color: #ffffff;
+    padding: 10px;
+}
```
Comment (BoshraM) [inline client/src/styles/MilestoneTable.css:25]: good job as the other one we need to think about some css root and default for now its great

**Review by BoshraM — APPROVED** [review]

File: client/src/component/MilestoneTable.js (hunk 1 above)
Comment (Zobeir-Rigi) [reply client/src/component/MilestoneTable.js:4]: using underscore( _ ) is not suggested in class naming, cool we can talk about it too

Comment (PeteLindsell) [reply client/src/component/MilestoneTable.js:4]: Yes we should talk as a team about css. Just remember that wireframes are not designs so you shouldnt be spending time making the design of the MVP match the wireframe

File: client/src/styles/MilestoneTable.css [hunk 4]
```diff
@@ -0,0 +1,25 @@
+.table-wrapper {
+    display: flex;
+    width: 100%;
+    height: 100px;
+    justify-content: center;
+    align-items: center;
+    border-radius: 7px;
+    background-color: #f5f5f5;
+/* #eef4f9 */
```
Comment (PeteLindsell) [inline client/src/styles/MilestoneTable.css:9]: Make sure you take a look through the pr and remove any unused/unwanted code

**Review by PeteLindsell — COMMENTED** [review]
Looks good 👍 just a few comments bellow

File: client/src/styles/MilestoneTable.css (hunk 4 above)
Comment (Zobeir-Rigi) [reply client/src/styles/MilestoneTable.css:9]: Just to wanted save this color as I like it, cool, nice point

**Pull request merged**